package.json

```diff
@@ -4,9 +4,8 @@
   "description": "Netlify Build plugin - {{description}}",
   "main": "src/index.js",
   "files": [
-    "src",
-    "manifest.yml",
-    "!*~"
+    "src/**/*.js",
```

Comment:
Wondering if this may be a problem for plugins that have some sort of templates or non `js` files that need to be published 🤔 (i.e. https://github.com/racoonx2p/netlify-plugin-gmail/tree/master/src/templates). A super specific case probably and I don't really have a better alternative, so maybe just something for us keep an eye out for when reviewing plugins?

Reply:
Great point. I've added more file extensions in 28f20746235b1b21575e683d2f0bde92001dc6d1